test/e2e/flowaggregator_test.go

```diff
@@ -22,94 +22,98 @@ import (
+    reversePacketDeltaCount: 0
+    reverseOctetDeltaCount: 0
     sourcePodName: perftest-a
     sourcePodNamespace: antrea-test
     sourceNodeName: k8s-node-master
     destinationPodName: perftest-b
     destinationPodNamespace: antrea-test
     destinationNodeName: k8s-node-master
-    destinationClusterIPv4: 10.103.234.179
-    destinationServicePortName: antrea-test/perftest-b:
-    ingressNetworkPolicyName: test-networkpolicy-ingress
+    destinationServicePort: 5201
+    destinationServicePortName:
+    ingressNetworkPolicyName: test-flow-aggregator-networkpolicy-ingress
     ingressNetworkPolicyNamespace: antrea-test
-    egressNetworkPolicyName: test-networkpolicy-egress
+    egressNetworkPolicyName: test-flow-aggregator-networkpolicy-egress
     egressNetworkPolicyNamespace: antrea-test
+    destinationClusterIPv4: 0.0.0.0
+    originalExporterIPv4Address: 10.10.0.1
+    originalObservationDomainId: 2134708971
+    octetDeltaCountFromSourceNode: 0
+    octetTotalCountFromSourceNode: 23459802093
+    packetDeltaCountFromSourceNode: 0
+    packetTotalCountFromSourceNode: 537924
+    reverseOctetDeltaCountFromSourceNode: 0
+    reverseOctetTotalCountFromSourceNode: 23108308
+    reversePacketDeltaCountFromSourceNode: 0
+    reversePacketTotalCountFromSourceNode: 444320
+    octetDeltaCountFromDestinationNode: 0
+    octetTotalCountFromDestinationNode: 23459802093
+    packetDeltaCountFromDestinationNode: 0
+    packetTotalCountFromDestinationNode: 537924
+    reverseOctetDeltaCountFromDestinationNode: 0
+    reverseOctetTotalCountFromDestinationNode: 23108308
+    reversePacketDeltaCountFromDestinationNode: 0
+    reversePacketTotalCountFromDestinationNode: 444320
 
 Intra-Node: Flow record information is complete for source and destination e.g. sourcePodName, destinationPodName
 Inter-Node: Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info e.g., sourcePodName, sourcePodNamespace, sourceNodeName etc.
 AntreaProxy enabled (Intra-Node): Flow record information is complete for source and destination along with K8s service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
 AntreaProxy enabled (Inter-Node): Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info like in Inter-Node case along with K8s Service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
 */
 
-func TestFlowExporter(t *testing.T) {
-	// Should I add skipBenchmark as this runs iperf?
+const (
+	ingressNetworkPolicyName   = "test-flow-aggregator-networkpolicy-ingress"
+	egressNetworkPolicyName    = "test-flow-aggregator-networkpolicy-egress"
+	collectorCheckDelay        = 1 * time.Second
+	expectedNumTemplateRecords = 1
+	// Single iperf run results in two connections with separate ports (control connection and actual data connection). Suspecting second flow to be control
```

Comment:
you can remove "Suspecting second flow to be control flow to exchange stats info." now

Reply:
Thanks, addressed.